emscripten.py

```diff
@@ -1833,12 +1833,12 @@ def build_wasm(temp_files, infile, outfile, settings, DEBUG):
     s2wasm_args = create_s2wasm_args(temp_s, wasm)
     if settings['DEBUG_LEVEL'] >= 2 or settings['PROFILING_FUNCS']:
       s2wasm_args += ['-g']
-      if settings['DEBUG_LEVEL'] >= 4:
-        s2wasm_args += ['--source-map=' + wasm + '.map']
-        if not settings['SOURCE_MAP_BASE']:
-          logging.warn("Wasm source map won't be usable in a browser without --source-map-base")
-        else:
-          s2wasm_args += ['--source-map-url=' + settings['SOURCE_MAP_BASE'] + os.path.basename(settings['WASM_BINARY_FILE']) + '.map']
+    if settings['DEBUG_LEVEL'] >= 4:
```

Comment:
I assume this is cleanup and not semantically meaningful? Because it should be a strict subset of the other condition.

I like this better (nesting bad! flat good!) but making sure I'm reading it correctly

Reply:
right